src/components/Header.js

```diff
@@ -1,95 +1,100 @@
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
 const Header = () => {
-  const { user ,setUser } = useContext(UserContext);
+  const { user, setUser } = useContext(UserContext);
   const isOnline = useOnline();
   const cart = useSelector((state) => state.cart.items);
   const recipes = useSelector((state) => state.recipe.recipes);
 
+
+const [isMenuOpen , setMenuOpen] = useState(false);
   const handleLogout = () => {
     setUser({
-        name: "Guest",
-        email: "email.com",
-        password: "password",
-        login: false,
+      name: "Guest",
+      email: "email.com",
+      password: "password",
+      login: false,
     });
-};
+  };
+
+  const toggleMenu = () => {
+    setMenuOpen(!isMenuOpen);
+  };
+
   return (
     <>
-    <div className=" flex justify-between bg-transparent shadow-md my-2 py-2 ">
-      <div className=" mx-4 my-2 ">
-        <Link href="/">
-          <img
-            src={Loco}
+      <div className="flex justify-between items-center bg-transparent shadow-md my-2 py-2 px-4 flex-wrap">
+        <div className="my-2">
+          <Link to="/">
+            <img
+              src={Loco}
               alt="logo"
               data-testid="logo"
-              className="h-12 w-12 shadow-sm  border  rounded-tr-lg rounded-bl-lg "
-          />
-        </Link>
-      </div>
-      <div >
-        <ul className="flex flex-row space-x-5 p-3 m-2" >
-          <li>
-            <Link to="/">Home</Link>
-          </li>
-          <li>
-            <Link to="/about">About</Link>
-          </li>
-          <li>
-            <Link to="/contact">Contact</Link>
+              className="h-12 w-12 shadow-sm border rounded-tr-lg rounded-bl-lg"
+            />
+          </Link>
+        </div>
+
+        {/* Hamburger Icon */}
+        <div className="md:hidden flex items-center" onClick={toggleMenu}>
+          <FontAwesomeIcon icon={faBars} size="2xl" />
+        </div>
+
+        {/* Navigation Links */}
+        <div className={`w-full md:w-auto ${isMenuOpen ? "block" : "hidden"} md:block`} >
+          <ul className="flex flex-col md:flex-row space-y-2 md:space-y-0 md:space-x-5 p-3 m-2">
+            <li >
+              <Link className="hover:text-pink-300 transform transition-transform duration-300 scale-100 hover:text-xl" to="/">Home</Link>
+            </li>
+            <li>
+              <Link className="hover:text-pink-300 transform transition-transform duration-300 scale-100 hover:text-xl" to="/about">About</Link>
             </li>
             <li>
-              <Link className="flex" to="/recipeStore">
-              <img src={RecipeStore} alt="recipeStore" className="h-5 w-5 mx-2" /> -
-               <span className="mx-2">{recipes.length}</span>
+              <Link className="hover:text-pink-300 transform transition-transform duration-300 scale-100 hover:text-xl" to="/contact">Contact</Link>
```

Comment:
on hover not need to change test size , only make highlight

Reply:
okay